_articles/network.md

```diff
@@ -0,0 +1,312 @@
+---
+layout: article
+title: Basic Networking Diagnosis
+description: Instructions on how to help diagnose networking issues.
+keywords:
+  - Support
+  - networking
+  - command line
+  - diagnosis
+image: http://support.system76.com/images/system76.png
+hidden: false
+section: software-applications
+
+---
+
+Finding out why your system is having network issues can be a daunting task.  We'll explore some useful commands 
+
+A quick review of the TCP/IP model
+---------------------------------------------
+
+First, let's take a moment to review the fundamentals of the TCP/IP network model.
+
+![TCP/IP OSI model](/images/network/tcpip.png)
+
+The layers in the TCP/IP network model, in order, include:
+
+-   **Layer 5:** Application
```

Comment:
Application is layer 7, not 5, and you skipped the Presentation (6) and Session (5) layers.

Reply:
I was going off of the OSI model

Reply:
The OSI model has seven layers: https://en.wikipedia.org/wiki/OSI_model

Reply:
5 layers is correct for TCP/IP, as it is typically discussed: https://en.wikipedia.org/wiki/Internet_protocol_suite#Comparison_of_TCP/IP_and_OSI_layering

Reply:
I see now that this section is labeled "in the TCP/IP network model," not OSI (although @linuxgnuru just said he was going off of OSI.) For TCP/IP, shouldn't the bottom two layers be collapsed into one as well? The TCP/IP model doesn't separate data link from physical.

https://bytesofgigabytes.com/networking/tcp-ip-model/ 
https://www.guru99.com/tcp-ip-model.html 
https://www.geeksforgeeks.org/tcp-ip-model/

Reply:
It seems there's some debate about what model to use for TCP/IP, but 5 layers is fairly common. Physical layer would be, say, 1000BASE-T over twisted pair copper wire, while the link layer deals in Ethernet frames that are independent of the exact physical medium.

The layers in a modern TCP/IP stack aren't all entirely independent.

Reply:
While I acknowledge some authors write five layers in TCP/IP-like models, the Wikipedia page you linked seems to describe four layers:

![image](https://user-images.githubusercontent.com/7199422/121032455-83ff3980-c768-11eb-8a4c-32b4059e5655.png)

![image](https://user-images.githubusercontent.com/7199422/121032547-98dbcd00-c768-11eb-9297-f32008e95f37.png)

The page states that "Several authors have attempted to incorporate the OSI model's layers 1 and 2 into the TCP/IP model... This often results in a model with five layers," but also that "The IETF protocol development effort is not concerned with strict layering." and that "Some of the networking models are from textbooks, which are secondary sources that may conflict with the intent of RFC 1122 and other IETF primary sources."

This entire support article seems somewhat out-of-scope to me (I somehow doubt that someone who doesn't want to research these topics from more authoritative sources would take the time to read and understand the content here), but if it's going to be here, I would personally recommend using the OSI model since it's actually definitive. I suppose since the TCP/IP model isn't very well defined, I can't win an argument about the number of layers being incorrect, but it looks off to me.

The [image](https://github.com/system76/docs/tree/network/images/network) that @linuxgnuru included also lists four layers for the TCP/IP model, so if we're taking the stance that it's five layers, that graphic conflicts with the text of our article.

The next line, "We will assume that you are familiar with this model," also seems strange. What happens if the reader's not familiar with the model?

Reply:
The image has been removed and the "assuming" line has been removed, so I'm marking this as resolved.